src/components/Toggle/Toggle.tsx

```diff
@@ -64,7 +66,7 @@ function Toggle({
       layout={layout}
       id={id}
       error={error}
-      align={layout === 'vertical' && align}
+      align={layout === 'vertical' ? align : undefined}
```

Comment:
this might break the layout for Toggle - i can have a look and check.

Reply:
`layout === 'vertical' && align` actually returns `false` when `layout` is not `vertical`, so typescript throws an error because type of `align` in FormLayout is `string | undefined`. 
`layout === 'vertical' ? align : undefined` This returns undefined if `layout` isn't `vertical`.

I also tested the alignment in storybook and the left right and undefined works fine when layout is vertical. (and do nothing when layout is horizontal or undefined)

Nothing really changes as align was only being set as `'left'` `'right'` when layout was vertical and this is doing the same thing except it sets align as undefined when `layout` is not `'vertical'`, while earlier it was being set to false